folx/wrapped_functions.py

```diff
@@ -293,11 +313,11 @@ def real_wrapper(
         flags=FunctionFlags.INDEXING | FunctionFlags.SCATTER,
         name='scatter',
     ),
-    jax.lax.scatter_add_p: wrap_forward_laplacian(
-        jax.lax.scatter_add_p.bind,
-        flags=FunctionFlags.LINEAR | FunctionFlags.SCATTER,
-        name='scatter_add',
-    ),
+    # jax.lax.scatter_add_p: wrap_forward_laplacian(
```

Comment:
Remove dead code

Reply:
That code is somewhat intended to be temporary until I rewrite the scatter implementation. At the moment, it's simply slower than doing the naive thing and densifying everything. Once scatter is properly implemented that may change.

Should I still remove it or just add a comment?

Reply:
ah if that was intentional that's fine - maybe add this as a comment line above so that it doesn't look like just dead code?

Reply:
done